bin/touch_mend

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+
+require "bundler/inline"
+gemfile do
+  source "https://rubygems.org"
+  gem "multi_repo", require: "multi_repo/cli"
+end
+
+opts = Optimist.options do
+  opt :branch, "The branch name.", :type => :string, :default => "master"
+
+  MultiRepo::CLI.common_options(self, :repo_set_default => nil)
+end
+opts[:repo_set] = opts[:branch] unless opts[:repo] || opts[:repo_set]
+
+MultiRepo::CLI.each_repo(**opts) do |repo|
+  next unless repo.git.remote_branch?("origin", opts[:branch])
+
+  repo.git.fetch
+  repo.git.hard_checkout(opts[:branch])
+
+  mend_file = repo.path.join(".whitesource")
+  next unless mend_file.exist?
+
+  mend_file.write(mend_file.read << "\n")
```

Comment:
🤦🏻‍♂️ It's sad that we need to do this... Would it be better to add our own key to the JSON and bump the value when needed?

```json
{
  "ManageIQTriggerScan": 1
}
```

Reply:
My only concern there is an invalid keys may break Mend.  Yeah I don't love that that's how the tool works :(